README.md

```diff
@@ -54,8 +54,7 @@ point that out explicitly and clearly in the associated patches and Cc
   fully correct: what udevd actually should be watching is the locks
   being released, not the devices being closed.
 
-* `SCM_CGROUP` or a similar auxiliary socket message, that allows
-  receivers to figure out which cgroup a sender is part of.
+* Expose cgroup id in a pidfd's `/proc/<pid>/fdinfo/<nr>`
```

Comment:
i still have wishes for SCM_CGROUP, i.e. cgroup of sender at the precise time it sent the message would be very useful. 

I'll take cgroup info via fdinfo any day, but it's not quite as great as SCM_CGROUP because one needs to be so very careful not to forget that there might be time-to-use/time-to-check issues here if the reading of cgroup membership is not done at time the message was originally sent.

I have no illusions that SCM_CGROUP is coming any time soon, but it is certainly still on my wishlist...

Reply:
I think that SCM_CGROUP is especially nasty. Once you have the ability to receive a pidfd and we introduced struct pidfd_info instead of just stashing struct pid figuring out the cgroup on the receiving side is trivial and doesn't require SCM_CGROUP anymore. So there's no need to have both options. It's also not a great fit into this api in my opinion.

Reply:
How would cgroup id help with the use case? (IIUC, journald would have to somehow remember id->cgroup path mapping to be useful.)
I'm not convinced about the necromancy in either form.